src/render.rs

```diff
@@ -54,6 +56,22 @@ pub const fn next_multiple_of(val: u32, rhs: u32) -> u32 {
     }
 }
 
+// This must be kept in sync with the struct in shaders/shared/bump.wgsl
```

Comment:
I think the "inverse" of this in `shader/shared/bump.wgsl` is missing 

Also:
```suggestion
// This must be kept in sync with the struct in shader/shared/bump.wgsl
```

Reply:
It is missing. Will fix.